README.md

```diff
@@ -72,26 +77,77 @@ to execute the following step one time.
 GO111MODULE=on go get github.com/jteeuwen/go-bindata/go-bindata@v3.0.8-0.20180305030458-6025e8de665b
 ```
 
-To build the operator, simply:
+#### Clone the repo
 
 ```bash
 git clone https://github.com/istio/operator.git
 cd operator
+```
+
+#### CLI
+
+To build the operator CLI, simply:
+
+```bash
 make mesh
 ```
 
 This will create a binary called `mesh` in ${GOPATH}/bin. Ensure this is in your PATH to run the examples below.
 
-### Flags
+#### Controller
+
+Building a custom controller requires a Dockerhub (or similar) account. To build using the container based build:
+
+```bash
+HUB=docker.io/<your-account> TAG=latest make docker.all
+```
+
+This builds the controller binary and docker file, and pushes the image to the specified hub with the `latest` tag.
+
+Note: The container based build does not work in all setups due to security restrictions. If the above does not work for you,
```

Comment:
should work in gcr.io, docker.io, and also now works in google's deb10 distro as well as ubuntu 16/18 and macos.  If you find someone struggling to push images plmk- otherwise i think this can be deleted.

(the work to make this so just hit the PR queue this morning).

Reply:
pretty much anyone in google cannot make it work because of the docker in docker build issue. both @howardjohn and i tried to make it work without success.

Reply:
yes.  make sure to add your uid to the docker group as per docker documentation.

Reply:
Verified this works now. Thanks!